spring-cloud-starter-stream-source-sftp/src/main/java/org/springframework/cloud/stream/app/sftp/source/tasklauncher/SftpSourceTaskLauncherConfiguration.java

```diff
@@ -53,9 +53,9 @@
 
 	protected static final String SFTP_PASSWORD_PROPERTY_KEY = "sftp_password";
 
-	protected static final String DATASOURCE_URL_PROPERTY_KEY = "spring_datasource_url";
+	protected static final String DATASOURCE_URL_PROPERTY_KEY = "spring.datasource.url";
 
-	protected static final String DATASOURCE_USERNAME_PROPERTY_KEY = "spring_datasource_username";
+	protected static final String DATASOURCE_USERNAME_PROPERTY_KEY = "spring.datasource.username";
```

Comment:
the `sftp_*` properties seem to be mapped fine into my `ConfigurationProperties` class from my testing in a sample app. since these are termed "environmentProperties" I do like the `_` as they are more env var looking, but if you would still like them changed i can do so.

Reply:
Ok. Fair enough. Fine with me. Merging...